website/docs/r/organization_run_task.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_organization_run_task"
+sidebar_current: "docs-resource-tfe-organization-run-task"
+description: |-
+  Manages Run tasks.
+---
+
+# tfe_organization_run_task
+
+[Run tasks](https://www.terraform.io/cloud-docs/workspaces/settings/run-tasks) allow Terraform Cloud to interact with external systems at specific points in the Terraform Cloud run lifecycle. Run tasks are reusable configurations that you can attach to any workspace in an organization
+
+The tfe_organization_run_task resource creates, updates and destroys Run tasks.
```

Comment:
I would rephrase this. Typically we give a little more context about the resource if necessary. In this case, we should make the logical distinction between what a Workspace and Organization run task are.

Reply:
So rather than that, I've adding links to our documentation as it's always up to date and there's a lot of context there which is too big for the provider docs.